ocs_ci/deployment/flexy.py

```diff
@@ -160,6 +159,45 @@ def deploy_prereq(self):
                         ),
                     }
                 )
+                logger.info(
+                    "Configured parameters in LAUNCHER_VARS for Proxy cluster: "
+                    f"{config.FLEXY['LAUNCHER_VARS']}"
+                )
+            if config.DEPLOYMENT.get("disconnected"):
+                base_domain = config.ENV_DATA["base_domain"]
+                cluster_name = config.ENV_DATA["cluster_name"]
+                cluster_domain = f".{cluster_name}.{base_domain}"
+                config.FLEXY["LAUNCHER_VARS"].update(
+                    {
+                        "http_proxy": config.ENV_DATA["disconnected_http_proxy"],
+                        "https_proxy": config.ENV_DATA.get(
+                            "proxy_https_proxy",
+                            config.ENV_DATA["disconnected_http_proxy"],
+                        ),
+                        "no_proxy": f"{cluster_domain},ocs-qe-proxy-dc-eco.usersys.redhat.com",
```

Comment:
might this should be f"{cluster_domain}.ocs-qe-proxy-dc-eco.usersys.redhat.com" ?

Reply:
It is a comma separated list of URLs - first URL is the cluster domain, the second is mirror registry URL, but it shouldn't be hardcoded here - I'll send a fix.